Support for GNU/kFreeBSD, by Aurelien Jarno.

--- libpcap0.8-0.9.7.orig/configure
+++ libpcap0.8-0.9.7/configure
@@ -5217,7 +5217,7 @@
 #define HAVE_CLONING_BPF 1
 _ACEOF
 
-elif test -r /dev/bpf0 ; then
+elif test -r /dev/bpf0 || test -r /usr/include/net/bpf.h ; then
 	V_PCAP=bpf
 elif test -r /usr/include/net/pfilt.h ; then
 	V_PCAP=pf
--- libpcap0.8-0.9.7.orig/configure.in
+++ libpcap0.8-0.9.7/configure.in
@@ -190,7 +190,7 @@
 	#
 	V_PCAP=bpf
 	AC_DEFINE(HAVE_CLONING_BPF,1,[define if you have a cloning BPF device])
-elif test -r /dev/bpf0 ; then
+elif test -r /dev/bpf0 || test -r /usr/include/net/bpf.h ; then
 	V_PCAP=bpf
 elif test -r /usr/include/net/pfilt.h ; then
 	V_PCAP=pf
