Support for GNU/kFreeBSD, by Aurelien Jarno.

--- libpcap0.8-0.9.5.orig/configure
+++ libpcap0.8-0.9.5/configure
@@ -5206,7 +5206,7 @@
 echo $ECHO_N "checking packet capture type... $ECHO_C" >&6
 if test ! -z "$with_pcap" ; then
 	V_PCAP="$withval"
-elif test -r /dev/bpf0 ; then
+elif test -r /dev/bpf0 || test -r /usr/include/net/bpf.h ; then
 	V_PCAP=bpf
 elif test -r /usr/include/net/pfilt.h ; then
 	V_PCAP=pf
--- libpcap0.8-0.9.5.orig/configure.in
+++ libpcap0.8-0.9.5/configure.in
@@ -184,7 +184,7 @@
 AC_MSG_CHECKING(packet capture type)
 if test ! -z "$with_pcap" ; then
 	V_PCAP="$withval"
-elif test -r /dev/bpf0 ; then
+elif test -r /dev/bpf0 || test -r /usr/include/net/bpf.h ; then
 	V_PCAP=bpf
 elif test -r /usr/include/net/pfilt.h ; then
 	V_PCAP=pf
